sites/public/styles/overrides.scss

```diff
@@ -349,10 +349,12 @@ a.button.is-primary:hover {
 }
 
 .hero__button__first {
-  margin-bottom: 1rem;
+  background-color: white;
+  width: 210px;
```

Comment:
Instead of hardcoding a specific width we should set `width: auto` so it scales with the text and will work with other languages. It looks like the mocks want padding of 14px top/bottom and 36px left/right so do ` padding: 0.875rem 2.25rem`. Since this is the same for both buttons we can put that on .hero__button